### PR TITLE
Updated windows guide to recommend prompt-toolkit  instead of pyreadl…

### PR DESCRIPTION
**Pull request by jsarver**

### CODE REVIEW

File: docs/windows.rst [hunk 1]
```diff
@@ -12,7 +12,16 @@ Next, install the other dependencies via ``pip``:
...
+   > pip install prompt-toolkit
+
+While Prompt-toolkit is considered an optional dependency, it's the
+recommended alternative to pyreadline for windows users.
+Once installed, you have to enable prompt-toolkit by adding the following to
+your ``~/.xonshrc`` file:
+
+.. code-block:: bat
+
+$SHELL_TYPE = 'prompt_toolkit'
```
Comment (scopatz) [inline docs/windows.rst]: Doesn't this need to be indented to be part of the code block?

File: docs/windows.rst [hunk 2]
```diff
@@ -12,7 +12,16 @@ Next, install the other dependencies via ``pip``:
...
    > pip install ply
-   > pip install pyreadline
+   > pip install prompt-toolkit
+
+While Prompt-toolkit is considered an optional dependency, it's the
+recommended alternative to pyreadline for windows users.
+Once installed, you have to enable prompt-toolkit by adding the following to
+your ``~/.xonshrc`` file:
+
+.. code-block:: bat
```
Comment (scopatz) [inline docs/windows.rst]: You can change the language to `xonsh`, actually.

**Comment by scopatz** [comment]
Thanks for putting this in @jsarver!

**Comment by jsarver** [comment]
Whoops, I'll fix those and resend.

**Comment by jsarver** [comment]
Hi @scopatz, it should be fixed now, sorry for the typos.

**Comment by scopatz** [comment]
Thanks a million @jsarver!

**Pull request merged**